testem.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  disable_watching: true,
```

Comment:
I'm not a fan of this file being changed like this. The properties were sorted alphabetically and as far as I can tell for the most part this change just scrambles them into an unsorted manner.